server/pbench/bin/unittests

```diff
@@ -81,6 +82,10 @@ function _run_activate {
     oldpath=$PATH
     PATH=$_tdir:$PATH
     $_tdir/pbench-server-activate ${_testdir}/opt/pbench-server/lib/config/pbench-server.cfg ${_testdir} >> $_testout
+    echo "--- Finished activate for $test (status=$?)" >> $_testout
+    echo "+++ ${_testdir}/opt/pbench-server/lib/crontab/crontab" >> $_testout
+    grep -v PATH= ${_testdir}/opt/pbench-server/lib/crontab/crontab >> $_testout
```

Comment:
Can you add a comment for why we ignore the `PATH` variable setting?   One guess would be that it is because it is different between tests.  Could we consider searching for the expected "full" `PATH` variable value to skip that, so that if the path is not what is expected, then `grep` command won't match, allowing the unit tests will fail with that path information?

Reply:
Reworked the mailto/mailfrom settings in the `pbench-server-activate-create-crontab` script and simplified the structure by getting rid of unneeded generality. Reworked the unittests script to simplify it and avoid some pain points. PTAL and let me know.